apps/www/components/Frame/Header.js

```diff
@@ -8,8 +8,8 @@ import {
   HeaderHeightProvider,
   useColorContext,
   shouldIgnoreClick,
+  BackIcon,
 } from '@project-r/styleguide'
```

Comment:
It was a bit hard to understand that the Toggle now is the AudioPlayer-Toggle. Maybe rename?